15-foundations-applications.Rmd

```diff
@@ -24,6 +24,32 @@ inference_method_summary_table %>%
   add_header_above(c("", "Answer" = 3))
 ```
 
+You might have noticed that the word *distribution* is used throughout this part (and will continue to be used in future chapters).
+A distribution always describes variability, but sometimes it is worth reflecting on **what** is varying.
+Typically the distribution either describes how the observations vary or how a statistic varies.
+But even when describing how a statistic varies, there is a further consideration with respect to the experimental design (e.g., does the statistic vary from random sample to random sample? or does it vary from random allocation to random allocation?).
+The methods presented in this text (and used in science generally) are typically used interchangeably across ideas of random samples or random allocation of the treatment.
+Oftentimes, two different analysis methods will give equivalent conclusions.
+The most important thing to consider, is how to contextualize the conclusion in terms of the problem.
+See Figure \@ref(fig:randsampValloc) to confirm that your conclusions are appropriate.
+
+Below, we synthesize the different types of distributions discussed throughout the text.
+Reading through the different definitions and solidifying your understanding will help as the distributions are used in future chapters.
+
+::: {.important data-latex=""}
+**Distributions.**
+
+-   A **data distribution** describes the shape, center, and variability of the **observed data**.
+
+-   A **population distribution** describes the shape, center, and variability of the entire **population of data**.
+
+-   A **sampling distribution** describes the shape, center, and variability of all possible values of a **sample statistic** from samples of a given sample size from a given population.
+
+-   A **randomization distribution** describes the shape, center, and variability of all possible value of a **sample statistic** from random allocations of the treatment variable.
```

Comment:
```suggestion
-   A **randomization distribution** describes the shape, center, and variability of all possible values of a **sample statistic** from random allocations of the treatment variable. We computationally generate the randomization distribution, though usually, it's not feasible to generate the full distribution of all possible values of the sample statistic, so we instead generate a large number of them.
```

Reply:
I don't love the phrasing at the end of the second sentence, but I couldn't come up with something better.

Reply:
i can't figure out github.  so i'm going to add a suggestion here (sorry to make more work for you, but i figure it's better than messing everything up!).  i'm fine with the phrasing.  but i was thinking we would add one more thing:

Almost always, by randomly allocating the treatment variable, we create a distribution where the null hypothesis is true.